scheduler/reconcile_util.go

```diff
@@ -225,8 +225,20 @@ func (a allocSet) filterByTainted(taintedNodes map[string]*structs.Node, support
 	ignore = make(map[string]*structs.Allocation)
 
 	for _, alloc := range a {
```

Comment:
Do functions like this ever make you really really wish go had exhaustive pattern matching? 😀

Reply:
💯